Essentials/src/messages_de.properties

```diff
@@ -465,6 +465,9 @@ seenOffline=\u00a76Spieler\u00a7c {0} \u00a76ist \u00a74offline\u00a76 seit \u00
 seenOnline=\u00a76Spieler\u00a7c {0} \u00a76ist seit \u00a7c{1}\u00a7a online.
 sellBulkPermission=\u00a76Du hast keine Permission, in Massen zu verkaufen.
 sellHandPermission=\u00a76Du hast keine Permission, einzeln zu verkaufen.
+cannotSellNamedItem=\u00a76You are not allowed to sell named items.
+cannotSellThisNamedItem=\u00a76You are not allowed to sell this named item: \u00a74{0}
```

Comment:
Thanks for adding this message, but since you changed it to only display a list at the end using `cannotSellTheseNamedItems`, `cannotSellThisNamedItem` is now redundant 😕